source/gameengine/Physics/Bullet/CcdPhysicsController.h

```diff
@@ -768,6 +768,16 @@ class CcdPhysicsController : public PHY_IPhysicsController
 		m_cci.m_radius = margin;
 	}
 
+	// Cone Shape (radius & height)
+	virtual void SetConeShape(float radius, float height)
+	{
+		if (m_collisionShape && (m_collisionShape->getShapeType() == CONE_SHAPE_PROXYTYPE)) {
+			btConeShape *coneShape = static_cast<btConeShape *>(m_collisionShape);
+			coneShape->setRadius(radius);
```

Comment:
The unique recomendation that Erwin Coumans said me was: "Just remember to remove the rigid body from the world before making those changes, and then re-insert it."
I will modify the patch to follow it